engine/class_modules/priest/sc_priest.cpp

```diff
@@ -603,7 +603,14 @@ struct ascended_eruption_t final : public priest_spell_t
   double composite_aoe_multiplier( const action_state_t* state ) const override
   {
     double cam = priest_spell_t::composite_aoe_multiplier( state );
-    return cam / std::sqrt( state->n_targets );
+    int targets = state->n_targets;
+
+    if ( priest().options.priest_ascended_eruption_targets )
+    {
+      sim->print_debug( "{} {} sets damage multiplier as if it hit {} targets.", *player, *this, priest().options.priest_ascended_eruption_targets );
+      targets = priest().options.priest_ascended_eruption_targets;
```

Comment:
This should be += instead of =. The option targets are added to the actual ones to replicate a raid or group env.

Reply:
and it should always add that option in. By default, it should be 10 to emulate raids (majority of sims that are run)